src/control/server/storage/provider.go

```diff
@@ -666,6 +678,33 @@ func (p *Provider) UpdateBdevFirmware(req NVMeFirmwareUpdateRequest) (*NVMeFirmw
 	return p.bdev.UpdateFirmware(req)
 }
 
+// ValidateBdevConfig updates an existing SPDK bdev config, if necessary.
+func (p *Provider) ValidateBdevConfig(ctx context.Context, ctrlrs NvmeControllers) error {
```

Comment:
this name looks misleading, validate doesn't intuitively imply an update

Reply:
What do you suggest as an alternative verb? "Verify"? "UpdateIfNecessary" seems verbose.

Reply:
CheckOrUpdate?

Reply:
Is there anything wrong with "UpdateBdevConfig"? If no update is needed, it cheerfully returns nil and moves on.

Reply:
Sure, if that works for y'all I'll make that change.